### ISSUE TITLE
Irritating error message of the validator_client before genesis start.

### ISSUE THREAD

**Issue by Michael-A-Heuer** [sigp/lighthouse#1424]
## Description

Because the genesis of the medalla testnet has not started yet, the validator client writes an error message to the console. This irritated me because I assumed that something is set up the wrong way.

## Version

Branch: `master` (Lighthouse 0.1.2-a37e75f4)

## Present Behaviour

Currently, the `lighthouse validator_client` expects a running beacon chain node. 
Because the beacon chain node started with `lighthouse beacon_node --eth1 --http` is currently `Waiting for more deposits`,
this results in the following error message of the `lighthouse validator_client`:
```bash
Unable to connect to beacon node        error: "ReqwestError(reqwest::Error { kind: Request, url: \"http://localhost:5052/node/version\", source: hyper::Error(Connect, ConnectError(\"tcp connect error\", Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })) })"
```
## Expected Behaviour

I would expect an info instead of an error message saying something like 
`INFO Waiting for genesis.`

Additionally, the number of active validators could be displayed.

**Comment by ph1go** [comment]
Wait, is this expected behaviour? I've been tearing my hair out thinking I've been doing something wrong.

My beacon gets connection refused (see blow) when trying to connect to geth (which is fully synced) and I get the above in the validator.

`Jul 31 18:01:02.328 INFO Waiting for more deposits               valid_deposits: None, total_deposits: 0, min_genesis_active_validators: 16384, service: beacon
Jul 31 18:01:09.343 ERRO Failed to update eth1 deposit cache     error: GetBlockNumberFailed("Request failed: reqwest::Error { kind: Request, url: \"http://localhost:8545/\", source: hyper::Error(Connect, ConnectError(\"tcp connect error\", Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })) }"), service: beacon`

If that is how it's supposed to be then that is indeed a pretty unhelpful message...

**Comment by AgeManning** [comment]
Yes agreed this would be better UX. 

Some thought is required in how to do this correctly. The Validator client is very lightweight and does not contain information about chain state etc, it relies on the beacon node via the HTTP. 

For a number of reasons, we don't activate the HTTP server until genesis is known (a few days before a network launches). So adjusting this logic to accommodate validator clients running before a known genesis requires some thought.

**Comment by mgcrea** [comment]
Just encountered this as well and thought I had something badly configured.

Would be also nice to throttle the requests (maybe once every 30s) as it's a bit spammy.

**Comment by yowie** [comment]
🤦 i thought i had completely broken my install.

Can't wait to start!

**Comment by paulhauner** [comment]
This will be fixed alongside #1434 :)

**Comment by paulhauner** [comment]
#1862 should lay the groundwork to achieve this.
